cva6/regress/install-verilator.sh

```diff
@@ -9,12 +9,12 @@
 
 set -e
 
-if [ -z ${NUM_JOBS} ]; then
+if ! [ -n "${NUM_JOBS}" ]; then
```

Comment:
The objective of the change was to standardize the script. There are some places of the scripts where it is used instead of `-z`. To be honest I prefer too the other option so if it is fine for you then I would like to change all the places where `! [ -n`  is used to `-z`